### PR TITLE
Fix rerender

### LINKED ISSUE #192: Output of conda-smithy update-cb3 is not in the PR body
See https://github.com/conda-forge/boost-cpp-feedstock/pull/30

### LINKED ISSUE #193: Don't run the compiler migrator twice
See https://github.com/conda-forge/arrow-cpp-feedstock/pull/52/files

### PR DESCRIPTION
**Pull request by CJ-Wright**
close #193 
close #192

### CODE REVIEW

File: conda_forge_tick/migrators.xsh [hunk 1]
```diff
@@ -301,10 +301,14 @@ class Compiler(Migrator):
 
     def migrate(self, recipe_dir, attrs, **kwargs):
         self.out = $(conda-smithy update-cb3 --recipe_directory @(recipe_dir))
+        return True
 
     def pr_body(self):
         body = super().pr_body()
-        body.format('{}\n'
+        body = body.format('{}\n'
```
Comment (isuruf) [inline conda_forge_tick/migrators.xsh]: `.format(self.out)` should come here

Comment (CJ-Wright) [reply conda_forge_tick/migrators.xsh]: Will it properly do the string formatting?

**Comment by CJ-Wright** [comment]
@scopatz

File: conda_forge_tick/migrators.xsh (hunk 1 above)
Comment (isuruf) [reply conda_forge_tick/migrators.xsh]: Ignore my comment. It should work fine

**Pull request merged**